### ISSUE TITLE
Report Previewer: Up/Down arrows

### ISSUE THREAD

**Issue by npaszty** [insightsengineering/teal.reporter#107]
### What happened?

not sure what these arrows are supposed to do since clicking them didn't appear to do anything. though they might re-order the cords in the stack.  
![image](https://user-images.githubusercontent.com/40777373/177432441-38e88b70-ab2a-4195-8e1d-a4d9152741be.png)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct.

### Contribution Guidelines

- [X] I agree to follow this project's Contribution Guidelines.

### Security Policy

- [X] I agree to follow this project's Security Policy.

**Comment by Polkas** [comment]
Yes, they re-order the cards. If the neighbourhood cards are of the some module the process is not so noticeable now.
We already have issues for it:
- https://github.com/insightsengineering/teal.reporter/issues/35
- https://github.com/insightsengineering/teal.reporter/issues/94
